airsonic-main/src/main/java/org/airsonic/player/ajax/PlayQueueService.java

```diff
@@ -232,6 +242,20 @@ public PlayQueueInfo play(int id) throws Exception {
         return doPlay(request, player, songs).setStartPlayerAt(0);
     }
 
+    /**
+     * @param index Start playing at this index, or play whole radio playlist if {@code null}.
+     */
+    public PlayQueueInfo playInternetRadio(int id, Integer index) throws Exception {
+        HttpServletRequest request = WebContextFactory.get().getHttpServletRequest();
+        HttpServletResponse response = WebContextFactory.get().getHttpServletResponse();
```

Comment:
Why do you need the response?

Reply:
Ah, you're right, that's a copy-paste from other methods. Removed.